src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/ContentHelper.Common.cs

```diff
@@ -36,34 +36,9 @@ internal static string GetContentType(HttpResponseMessage response)
         }
 
         internal static Encoding GetDefaultEncoding()
-        {
-            return GetEncodingOrDefault((string)null);
-        }
-
-        internal static Encoding GetEncoding(HttpResponseMessage response)
-        {
-            // ContentType may not exist in response header.
-            string charSet = response.Content.Headers.ContentType?.CharSet;
-            return GetEncodingOrDefault(charSet);
-        }
-
-        internal static Encoding GetEncodingOrDefault(string characterSet)
-        {
-            // get the name of the codepage to use for response content
-            string codepage = (string.IsNullOrEmpty(characterSet) ? _defaultCodePage : characterSet);
-            Encoding encoding = null;
-
-            try
-            {
-                encoding = Encoding.GetEncoding(codepage);
-            }
-            catch (ArgumentException)
-            {
-                // 0, default code page
-                encoding = Encoding.GetEncoding(0);
-            }
-
-            return encoding;
+        { 
+            Encoding encoding = Encoding.GetEncoding(_defaultCodePage);
```

Comment:
We can remove `_defaultCodePage` at all since it is used only here:
```suggestion
            // default codepage encoding for web content.  See RFC 2616.
            Encoding encoding = Encoding.GetEncoding("ISO-8859-1");
```

Reply:
Done

Reply:
Following the `_defaultCodePage` case, I also removed `s_contentTypeParamSeparator`